pyproject.toml

```diff
@@ -136,21 +136,23 @@ module = [
 ignore_missing_imports = true
 
 [tool.ruff]
+extend-exclude = [
+    "src/_nebari/template",
+    "home",
+    "__pycache__"
+]
+
+[tool.ruff.lint]
 select = [
-    "E",
-    "F",
-    "PTH",
+    "E",  # E: pycodestyle rules
```

Comment:
flyby: add helpful comments

Reply:
also select and ignore had to be moved to [tool.ruff.lint].  I was getting a warning saying the old location was deprectaed.